hydradx/tests/test_stableswap.py

```diff
@@ -126,9 +131,9 @@ def test_spot_price(token_a: int, token_b: int, token_c: int, token_d: int, amp:
 
     spot_price_final = swap_pool.price(tkns[i], "A")
 
-    if spot_price_initial > exec_price and (spot_price_initial - exec_price) / spot_price_initial > 10e-10:
+    if spot_price_initial > exec_price and (spot_price_initial - exec_price) / spot_price_initial > 1e-5:
```

Comment:
Is this implying that the execution price might be less than the initial spot price by up to 1/10,000th? Why is that?

I also tried removing those conditions from the test and running it, and it passed.

Reply:
I changed to 1e-5 while troubleshooting something and forgot to change it back. Changing it back now.